src/store/modules/imageUploadStore.js

```diff
@@ -0,0 +1,33 @@
+import UploadService from '../../services/UploadService'
+
+export const imageUploadStore = {
+    mutations: {
+        SET_IMAGEPATH(state, response) {
+            if(response.code == 500) {
+                this.state.upload_message = 'Error uploading the file, please try again'
+                alert(this.state.upload_message)
+            }
+           
+            if(response.code == 200) {
+                this.state.imagePath = newState.data.imagePath
+            }
+
+            else {
+                this.state.upload_message = 'Error uploading the file, please contact the administrator'
+                alert(this.state.upload_message)
+            }
+        }
+    },
+    actions: {
+        async upload_to_s3(context, file) {
+            alert('uploading')
```

Comment:
trying to show status update? perhaps another task. we can remove this from here